src/test/java/io/stargate/sgv2/jsonapi/api/v1/InsertIntegrationTest.java

```diff
@@ -801,25 +802,39 @@ private JsonNode createBigDoc(String docId, int minDocSize) throws IOException {
 
       boolean bigEnough = false;
 
-      // Since we add one property before loop, reduce max by 1
```

Comment:
Had to change generation of "big document" in a way that will not violate any of other size limitations than (for one case) max doc size -- specifically, number of properties per Object, per Doc, and (default) String value limit.

Those are limits will be verified with different tests.